playground/src/containers/app.tsx

```diff
@@ -431,6 +431,7 @@ export class App extends React.Component<AppProps, AppState>
         const activeWorkspace = this.state.workspaces[this.state.activeWorkspaceName];
         const contents = workspaceToShareContent(activeWorkspace, this.inputEditors);
         const data = await share(contents);
+
```

Comment:
This looks unrelated!

Reply:
Ah yes, there used to be some JS which was later removed again. Will remove the extra whitespace 👍